src/backend/tests/reimbursement-requests.test.ts

```diff
@@ -222,4 +228,29 @@ describe('Reimbursement Requests', () => {
       expect(prisma.reimbursement_Request.update).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('Delivered Tests', () => {
+    test('Mark as delivered fails for non admins', async () => {
+      await expect(
+        ReimbursementRequestService.markReimbursementRequestAsDelivered(wonderwoman, PopEyes.vendorId)
+      ).rejects.toThrow(new AccessDeniedException('Only the creator of the reimbursement request can mark as delivered'));
+    });
+
+    test('Mark as delivered fails for undefined ID', async () => {
```

Comment:
you have to mock the find unique here, thats why its failing on github, you would mock resolve it as null to show that it doesnt exist